TSPL.docc/LanguageGuide/Protocols.md

```diff
@@ -1604,9 +1608,9 @@ that tries to adopt `SomeClassOnlyProtocol`.
 ## Protocol Composition
 
 It can be useful to require a type to conform to multiple protocols at the same time.
-You can combine multiple protocols into a single requirement
+You can combine multiple protocols into a single *protocol composition type*
```

Comment:
The language reference calls this a "protocol composition type", but I think the compiler warnings call this a "protocol-constrained type" 🤔